packages/google-vertex/src/google-vertex-embedding-model.test.ts

```diff
@@ -1,148 +1,277 @@
-import { JsonTestServer } from '@ai-sdk/provider-utils/test';
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { TooManyEmbeddingValuesForCallError } from '@ai-sdk/provider';
+import { postJsonToApi } from '@ai-sdk/provider-utils';
 import { GoogleVertexEmbeddingModel } from './google-vertex-embedding-model';
 
-const dummyEmbeddings = [
-  [0.1, 0.2, 0.3, 0.4, 0.5],
-  [0.6, 0.7, 0.8, 0.9, 1.0],
-];
-const testValues = ['sunny day at the beach', 'rainy day in the city'];
+// Mock the postJsonToApi utility
+vi.mock('@ai-sdk/provider-utils', async importOriginal => {
+  const actual = await importOriginal<
+    typeof import('@ai-sdk/provider-utils')
+  >();
+  return {
+    ...actual,
+    postJsonToApi: vi.fn(),
+  };
+});
```

Comment:
ideally align with other embedding tests that go through msw / network level mocking to keep with existing patterns, e.g. `packages/openai/src/openai-embedding-model.test.ts`

Reply:
Done.